packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -86,10 +86,19 @@ function Breadcrumbs(props: SpectrumBreadcrumbsProps, ref: DOMRef) {
   if (childArray.length > visibleItems) {
     let rootItems = showRoot ? [childArray[0]] : [];
 
-    // TODO: replace with menu component
     let menuItem = (
       <BreadcrumbItem key="menu">
-        <Menu isDisabled={isDisabled}>{childArray}</Menu>
+        <MenuTrigger
+          isDisabled={isDisabled}>
+          <ActionButton
+            aria-label="…"
```

Comment:
Is this supposed to be "..."?

Reply:
that aria-label was added by @majornista  in https://github.com/adobe-private/react-spectrum-v3/pull/65/files#diff-e8c06dedc039000aa3c30f7291fee9bbR162 so I think its correct